pdns/ednscookies.hh

```diff
@@ -21,13 +21,58 @@
  */
 #pragma once
 #include "namespaces.hh"
+#include "iputils.hh"
+
+#define EDNSCOOKIESECRETSIZE 32
+#define EDNSCOOKIEOPTSIZE 24
```

Comment:
Just a nit but I would prefer the use of static constants in the class below, so that these values are scoped. We could also perhaps add a `static_assert` checking that the secret size is `crypto_shorthash_KEYBYTES*2`, since it's hex-encoded?